services/web-app/secure-dev-server.js

```diff
@@ -15,8 +15,10 @@ const { RunMode, getRunMode, loadEnvVars } = require('@carbon-platform/api/runti
 
 const { PORT } = loadEnvVars({ PORT: '443' })
 
+const hostname = 'localhost'
+
 const isDevMode = getRunMode() === RunMode.Dev
-const app = next({ dev: isDevMode })
+const app = next({ dev: isDevMode, hostname, port: PORT })
```

Comment:
If not explicitly set, what does hostname get defaulted to?

Reply:
it doesn't it just errors out. 
<img width="1310" alt="image" src="https://user-images.githubusercontent.com/40550942/154520036-1474c77e-3547-4cee-9480-0cf01dd925d0.png">
<img width="713" alt="image" src="https://user-images.githubusercontent.com/40550942/154520087-602dc64f-156c-465d-926a-5123559af03b.png">
This server is only meant to be used locally so setting it to localhost shouldn't be a problem

Reply:
Okay cool. Was just curious if it had some sort of interesting auto-default. Thanks!